shenyu-admin/pom.xml

```diff
@@ -313,6 +313,12 @@
             <artifactId>shenyu-discovery-zookeeper</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.shenyu</groupId>
+            <artifactId>shenyu-spring-boot-starter-client-springmvc</artifactId>
+            <version>${project.version}</version>
```

Comment:
This solution of introducing client jars feels too heavy and not elegant. It should be better to use a listener (such as org. apache. shenyu. admin. listener. Application Start Listener) to directly call the internal method to initiate the registration process after the admin is started.

Reply:
After thinking about it, the "sdk+annotation" solution is perfect.